ros2bag/ros2bag/verb/info.py

```diff
@@ -26,11 +26,38 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '-t', '--topic-name', action='store_true',
             help='Only display topic names.'
         )
+        parser.add_argument(
+            '-v', '--verbose', action='store_true',
+            help='Display request/response information for services'
+        )
+
+    def _is_service_event_topic(self, topic_name, topic_type) -> bool:
+
+        service_event_type_middle = '/srv/'
+        service_event_type_postfix = '_Event'
+
+        if (service_event_type_middle not in topic_type
+                or not topic_type.endswith(service_event_type_postfix)):
+            return False
+
+        service_event_topic_postfix = '/_service_event'
+        if not topic_name.endswith(service_event_topic_postfix):
+            return False
+
+        return True
 
     def main(self, *, args):  # noqa: D102
-        m = Info().read_metadata(args.bag_path, args.storage)
-        if args.topic_name:
-            for topic_info in m.topics_with_message_count:
-                print(topic_info.topic_metadata.name)
+        if args.topic_name and args.verbose:
+            print("Warning! You have set both the '-t' and '-v' parameters. The '-t' parameter "
+                  'will be ignored.')
```

Comment:
> why printing service event topics and general topics are exclusive?

This follows the specifications as originally displayed.  
Before introducing service support, rosbag2 only list all recorded topic name if '-t' is specified. Otherwise, detailed information of topics are listed.  So this implies mutually exclusive behavior.
Now we introduce '-v' parameter to show detailed information of services.  It cannot use above potential behavior. So we have to explicitly describe this mutually exclusive behavior.  

> instead, we could have -s option to print service with topics?

Yes, we can support this parameter in the next step.